python/examples/cyl-ellipsoid.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import division
+
+import meep as mp
+from meep.geom import Cylinder, epsilon, Ellipsoid, Vector3, Medium
+from meep.simulation import Mirror, Pml, Simulation, no_size
+from meep.source import GaussianSource, Source
+
```

Comment:
The `geom`, `simulation`, and `source` objects should all be loaded by default. It is cumbersome to import them one-by-one like this.